common/lib/dependabot/file_updaters/base.rb

```diff
@@ -25,6 +26,21 @@ def updated_dependency_files
         raise NotImplementedError
       end
 
+      def pr_message
+        Dependabot::PullRequestCreator::MessageBuilder.new(
```

Comment:
Slight smell nesting this inside the FileUpdater as it doesn't update any files, thoughts on just calling this class directly instead?

Reply:
That way also easier to pass the options in directly from the updater until we figure out how to dismantle the updater and move more stuff into core.

Reply:
SGTM, this was based on @jurre's  intuition that while it's _cleaner_ to instantiate this class in our closed-source wrapper, that doesn't leave any sense in the open-source `dependabot-core` how the components are linked together.

It's also plausible that the `FileUpdater` could share state from the update to build the PR message: e.g. parsing `node_modules/foo/package.json` instead of re-fetching `foo` from the registry via the `MetadataFinder`.

Reply:
@jurre : I've reverted the changes to `FileUpdater::Base` in https://github.com/dependabot/dependabot-core/pull/3191/commits/40a357565cc70b0fdc4cbf65f72ee82a5a17597b .

I dig this more, I think _feelepxyz digs this more, but it does leave the `MessageBuilder` as a bit of detached magic: WDYT?